### PR TITLE
Task4

### PR DESCRIPTION
**Pull request by Chmelya**

### CODE REVIEW

File: task4/FiguresLibrary/Circle.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FiguresLibrary
+{
+    public class Circle: Shape
+    {
+        double d;
```
Comment (miko96) [inline task4/FiguresLibrary/Circle.cs:9]: use property

File: task4/FiguresLibrary/Circle.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        public Circle(double d) : base("Circle")
+        {
+            this.d = d;
+        }
+
+        public override double GetArea()
+        {
+            return Math.PI * (d / 2) * (d / 2);
```
Comment (miko96) [inline task4/FiguresLibrary/Circle.cs:17]: just PI*d

File: task4/FiguresLibrary/IShapeAreaCalculator.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+using System.Collections.Generic;
+using System.Text;
+
+namespace FiguresLibrary
+{
+    public interface IShapeAreaCalculator
+    {
+        double Calculate(IEnumerable<Shape> shapes); // returns a sum of all shapes areas
+    }
+
```
Comment (miko96) [inline task4/FiguresLibrary/IShapeAreaCalculator.cs:11]: remove empty line

File: task4/FiguresLibrary/ShapeAreaCalculator.cs [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+namespace FiguresLibrary
+{
+    public class ShapeAreaCalculator : IShapeAreaCalculator
+    {
+        public double Calculate(IEnumerable<Shape> shapes)
+        {
+            double result = 0;
+
+            foreach (var s in shapes)
```
Comment (miko96) [inline task4/FiguresLibrary/ShapeAreaCalculator.cs:13]: use brackets
```
foreach...
{
}
```

File: task4/FiguresLibrary/Circle.cs [hunk 5]
```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FiguresLibrary
+{
+    public class Circle: Shape
+    {
+        public double d { get; }
```
Comment (vikariusik) [inline task4/FiguresLibrary/Circle.cs:9]: Please name properties Uppercase, and use a full name like diameter, width, height

File: task4/FiguresLibrary/Triangle.cs [hunk 6]
```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (vikariusik) [inline task4/FiguresLibrary/Triangle.cs:2]: image
Please clean unnecessary namespaces

**Pull request merged**